src/store/inputSlice.js

```diff
@@ -146,6 +147,7 @@ function computeDriveInput(state, action) {
   driveInput.steer = driveGamepad["RightStickX"] + getAxisFromKeys(pressedKeys, "ARROWLEFT", "ARROWRIGHT");
   driveInput.tankLeft = driveGamepad["LeftStickY"] + getAxisFromKeys(pressedKeys, "ARROWDOWN", "ARROWLEFT")
   driveInput.tankRight = driveGamepad["RightStickY"] + getAxisFromKeys(pressedKeys, "ARROWRIGHT", "ARROWUP");
+  driveInput.activeSuspension = driveGamepad["RightStickX"] + getAxisFromKeys(pressedKeys, "ARROWDOWN", "ARROWRIGHT");
```

Comment:
These button mappings (down and right) interfere with the other button mappings. We should change it to be something unique, check the existing button mappings to find some buttons that make sense.